### ISSUE TITLE
Removing bytes doesn't work

### ISSUE THREAD

**Issue by trietptm** [sebastianbiallas/ht#12]
The menu Edit -> delete (Ctrl+Del/Alt+D) of HT current latest release (ht-2.1.0-win32.exe) doesn't work.

**Comment by sebastianbiallas** [comment]
Yeah, it unfortunately only works for text files, not for binaries at the moment.
